### PR TITLE
Implement corrections from feedback

### PR DESCRIPTION
**Pull request by xgyrosx**

### CODE REVIEW

**Review by sjossi — CHANGES_REQUESTED** [review]
some commas still need to be fixed. also, there's a "seperation" that should be "separation". I'll join in on the branch

File: doc/article/technical_article.tex [hunk 1]
```diff
@@ -121,17 +121,19 @@
 
   The backend is written in Python with the help of the web application
   framework Django and the frontend with the Javascript UI framework Vue.js.
-  Both technologies were new to most team members, but have proven themselves
-  effective and learning them were ultimately a benefit to the outcome of the
-  project.
+  Both technologies were new to most team members, but all members have proven
+  themselves
+  %effective
```
Comment (lucachristen) [inline doc/article/technical_article.tex]: Why is this commented out? Remove it if not needed anymore

File: doc/article/technical_article.tex [hunk 2]
```diff
@@ -121,17 +121,19 @@
...
+  self driven and learning these skills was ultimately a benefit to the
```
Comment (lucachristen) [inline doc/article/technical_article.tex]: self-driven with a hyphen

File: doc/article/technical_article.tex [hunk 3]
```diff
@@ -140,18 +142,22 @@ \section{Introduction}
...
 
 \subsection{Objectives}
 The goal with trckr is to develop and distribute a time tracking web application
-that is easy to understand and use. The most important requirement is to
-require just a few steps to track all the required information. Only this will
-keep the user engaged and raise the accuracy of the provided data.
+that is easy to both understand and use. The most important non-functional
+requirement is to focus on a small amount of steps for users to track all the
+required information.
+%The most important requirement is to require just a few steps to track all the
```
Comment (lucachristen) [inline doc/article/technical_article.tex]: Remove commented out text

File: doc/article/technical_article.tex [hunk 4]
```diff
@@ -188,15 +194,20 @@ \subsection{Node.js}
 
 \subsection{Vue.js}
 Vue.js is a progressive framework for building user interfaces.\cite{vuejs}
-Vue.js is used for trckr mainly for its simplicity and the rather shallow
-learning curve it provides to unexperienced developers. The features that Vue.js
-provides allow the creation of data structures that can easily be displayed in
-on a website. This and the ability to easily make calls to the backend make it a
-good fit for trckr.
+Vue.js is used mainly for its simplicity and the rather shallow learning curve
+it provides unexperienced developers. The features that Vue.js provides allow
```
Comment (lucachristen) [inline doc/article/technical_article.tex]: Try not to use "prodives" twice in a row
Rather something like: "The features of Vue.js allow..."

File: doc/article/technical_article.tex [hunk 5]
```diff
@@ -188,15 +194,20 @@ \subsection{Node.js}
...
+the creation of data structures that can easily be displayed on a website. This
+and the ability to easily make calls to the backend make it a good fit for
+trckr.
 
 \section{Results}
-The architecture was kept fairly simple, while still providing a clear seperation of concerns as illustrated in figure \ref{fig:architecture}, using Django with PostgreSQL in the backend and a
-frontend based around Vue.js.
+The architecture was kept fairly simple, while still providing a clear
+seperation of concerns as illustrated in figure \ref{fig:architecture}, using
```
Comment (lucachristen) [inline doc/article/technical_article.tex]: Ms. Rodriguez wants us to write "Figure" instead of "figure"

File: doc/article/technical_article.tex [hunk 6]
```diff
@@ -188,15 +194,20 @@ \subsection{Node.js}
...
+Vue.js is used mainly for its simplicity and the rather shallow learning curve
+it provides unexperienced developers. The features that Vue.js provides allow
+the creation of data structures that can easily be displayed on a website. This
+and the ability to easily make calls to the backend make it a good fit for
+trckr.
 
 \section{Results}
-The architecture was kept fairly simple, while still providing a clear seperation of concerns as illustrated in figure \ref{fig:architecture}, using Django with PostgreSQL in the backend and a
-frontend based around Vue.js.
+The architecture was kept fairly simple, while still providing a clear
```
Comment (lucachristen) [inline doc/article/technical_article.tex:200]: Remove the comma

File: doc/article/technical_article.tex [hunk 7]
```diff
@@ -188,15 +194,20 @@ \subsection{Node.js}
...
+seperation of concerns as illustrated in figure \ref{fig:architecture}, using
+Django with PostgreSQL in the backend and a frontend based around Vue.js.
```
Comment (lucachristen) [inline doc/article/technical_article.tex]: You could already write "front-end" and "back-end" even though the figure has yet to be updated :-)

File: doc/article/technical_article.tex [hunk 8]
```diff
@@ -188,15 +194,20 @@ \subsection{Node.js}
...
+
+%TODO: baliddim: I don't think we can write figure instead of Figure. -> by
```
Comment (lucachristen) [inline doc/article/technical_article.tex]: What do you mean with "by design"?

**Review by lucachristen — COMMENTED** [review]
Have a look at my comments and try to fix the issues.

File: doc/article/technical_article.tex (hunk 1 above)
Comment (xgyrosx) [reply doc/article/technical_article.tex]: the word was marked by the teacher. It's a suggestion for an alternative sentence :)

**Pull request merged**